server/models/User.js

```diff
@@ -43,6 +49,10 @@ User.associate = models => {
   User.hasMany(models.ApiKey, { as: 'apiKeys' });
   User.hasMany(models.Document, { as: 'documents' });
   User.hasMany(models.View, { as: 'views' });
+  User.belongsTo(models.User, {
+    as: 'suspendedBy',
+    foreignKey: 'suspendedById',
+  });
```

Comment:
@tommoor any idea why this doesn't seem to work for me? I set a user ID and try to access from suspended user with `user. suspendedBy` but I get an `undefined` back for some reason

Reply:
Not sure, but I feel like `hasO``e would be a more accurate association – maybe that would fix it?